meilisearch/src/search/mod.rs

```diff
@@ -796,8 +796,10 @@ fn prepare_search<'t>(
                     let span = tracing::trace_span!(target: "search::vector", "embed_one");
                     let _entered = span.enter();
 
+                    let deadline = std::time::Instant::now() + std::time::Duration::from_secs(10);
```

Comment:
```suggestion
                    let deadline = std::time::Instant::now() + std::time::Duration::from_secs(3);
```

Shouldn’t it be 3s here as well?

Reply:
This one is for pure semantic search. I thought 10s was better in that case because there's no fallback to full-text search when the embedding fails.

Reply:
you seems ok for me then we can merge 👍